src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesSpec.scala

```diff
@@ -55,5 +56,39 @@ class ResourceRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest
     }
 
   }
+
+  it should "200 on list resource roles" in {
+    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceAction("run")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
+    val samRoutes = createSamRoutes(Map(resourceType.name -> resourceType))
+
+    Post(s"/api/resource/${resourceType.name}/foo") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.NoContent
+    }
+
+    Get(s"/api/resource/${resourceType.name}/foo/roles") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.OK
+      responseAs[Set[String]]
+    }
+  }
+
+  it should "404 on list resource roles when resource type doesnt exist" in {
+    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceAction("run")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("run")))), ResourceRoleName("owner"))
+    val samRoutes = createSamRoutes(Map(resourceType.name -> resourceType))
+
+    Get(s"/api/resource/doesntexist/foo/roles") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.NotFound
+    }
+  }
+
+  it should "200 and return an empty list on list resource roles" in {
```

Comment:
This test is identical to the `Get` part of `it should "200 on list resource roles" in`. Should either do something more interesting or one of the two should be removed.